psl-core/src/main/java/org/linqs/psl/util/MathUtils.java

```diff
@@ -155,4 +155,83 @@ public static BigInteger factorial(BigInteger number) {
 
         return result;
     }
+
+    /**
+     * Given a vector in cartesian coordinates,
+     * return the vector in a log scale.
+     */
+    public static double[] toLogScale(double[] vector, double logBase) {
```

Comment:
Avoid graphing terminology here.
It can be confusing, since we are not graphing.
See if you can find another way to describe it.